backend/local/fadvise_unix.go

```diff
@@ -0,0 +1,81 @@
+//+build linux
+
+package local
+
+import (
+	"io"
+	"os"
+
+	"github.com/rclone/rclone/fs"
+	"golang.org/x/sys/unix"
+)
+
+type fadvise struct {
+	o          *Object
+	fd         int
+	lastPos    int64
+	curPos     int64
+	windowSize int64
+}
+
+func (f *fadvise) sequential(limit int64) {
+	l := int64(0)
+	if limit > 0 {
+		l = limit
+	}
+	if err := unix.Fadvise(f.fd, f.curPos, l, unix.FADV_SEQUENTIAL); err != nil {
+		fs.Infof(f.o, "fadvise sequential failed on file descriptor %d: %s", f.fd, err)
```

Comment:
This should probably be `fs.Errorf` if we really want the user to see it, or `fs.Debugf` if we don't care.  I think probably the latter - what do you think?

Reply:
Changing that to `fs.Debugf`.